docs/domains/Domain.json

```diff
@@ -421,6 +421,11 @@
             "$ref": "#/definitions/ServerStatus"
           }
         },
+        "retryCount": {
+          "description": "Number of retries.",
```

Comment:
This description isn't clear enough. Is this the number of attempts that have failed? What kind of failures does it include? When is it reset?

Reply:
To save time on the back-and-forth, can you start a slack discussion for this?

More questions:

What is the interval between retries?  If greater than 0, is the interval configurable?

How does this interact with the operator 'domainPresenceFailureRetryMaxCount and domainPresenceFailureRetrySeconds'?  

What's the default?

How do you force the system to try again?

Reply:
It only works with domainPresenceFailureRetryMaxCount (default is 5), I don't see domainPresenceFailureRetrySeconds used anymore.

Reply:
We can force retry before the 2 minute interval by changing restart/introspect/image name in the spec (or after it exceeded the limit).   Other changes can happen in between the intervals (e.g. configmap, image with same name/tag)